src/utils/getChangeArrowDirection.tsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react'
+
+export const getChangeArrowDirection = (change: number) => change > 0 ? <>&#129121;</> : <>&#129123;</>
```

Comment:
Maybe add the colour number to a const called red and green, which can be read easier

Reply:
This is unicode id but looks very similar to a hexidecimal colour :)

Reply:
You don't need to return JSX here, you could just return the string. And you could type it directly in the source code rather than use some cryptic code point. E.g.

`... => change > 0 ? "🡡" : "🡣"`